keps/3899-remove-finalizers-with-strict-patch/README.md

```diff
@@ -0,0 +1,88 @@
+# KEP-3899: Remove finalizers using strict patch
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+Remove finalizers using strict patch.
+This would force every finalizer update to be handled individually, removing the possibility of
+race conditions.
+
+## Motivation
+
+Updates to the finalizers [could be lost if they were made at the time when
+Kueue removed its system finalizers](https://github.com/kubernetes-sigs/kueue/issues/3899). 
+This could result in the unexpected and faulty behavior of the resources. 
+To prevent this, we need to use strict mode that ensures the concurrently-safe finalizer handling.
+
+### Goals
+
+- Remove finalizers using a strict patch.
+
+
+### Non-Goals
+
```

Comment:
```suggestion

- Implement a feature to provide a knob if Kueue uses strict mode or not.
```

it would be better obviously mention not to provide such a knob and we will remove non strict patch after this graduate to GA.

Reply:
Done, thanks for pointing out